app/src/main/java/org/simple/clinic/settings/changelanguage/ChangeLanguageEffectHandler.kt

```diff
@@ -10,13 +10,15 @@ object ChangeLanguageEffectHandler {
 
   fun create(
       schedulersProvider: SchedulersProvider,
-      settingsRepository: SettingsRepository
+      settingsRepository: SettingsRepository,
+      uiActions: UiActions
```

Comment:
`UiActions` interesting!

Reply:
No specific reason, I added this here because it was the most convenient. I'll move this out.